salt/modules/libcloud_loadbalancer.py

```diff
@@ -380,3 +452,17 @@ def _simple_member(member):
         'balancer': _simple_balancer(member.balancer),
         'extra': member.extra
     }
+
+
+def _sanitize_kwargs(kwargs):
```

Comment:
We should already have a function for this in `salt.utils` called `clean_kwargs`. Maybe using that one over implementing a new version would be good here.